internal/service/alarm_subscription_handler.go

```diff
@@ -300,11 +286,12 @@ func (h *alarmSubscriptionHandler) getSubcriptionId() (subId string) {
 // add it for now for the test purpose
 func (h *alarmSubscriptionHandler) encodeSubId(ctx context.Context,
 	subId string, input data.Object) (output data.Object, err error) {
-	//get cluster name, subscriptions
+	//get customer name, subscriptions
```

Comment:
Maybe you meant "consumer" instead of "customer"?

Reply:
Yes. changed thanks